### ISSUE TITLE
Engage link option

### ISSUE THREAD

**Issue by jimkp** [kororaproject/website#60]
The link to post option on right click in Engage doesn't show anything.

Ideally it should provide a pop-up with the link

**Comment by dyoungwd** [comment]
what link do you mean?

**Comment by jimkp** [comment]
Sorry it isn't right click. My mistake.
It is the option menu on the gear icon, see the attached screenshot.
![screenshot_2017-05-28_09-00-01](https://cloud.githubusercontent.com/assets/3682973/26524813/39e80a14-4384-11e7-8fd6-e98fe5f62a6d.png)

**Comment by jimkp** [comment]
I should this is in Chrome. I haven't tested in other browsers.

**Comment by dyoungwd** [comment]
yeah see what you mean for the "link" link - same on firefox and vivaldi.  You can right-click and copy the link then paste it in to the reply box no problem.

To get a popup perhaps editing /lib/Canvas/Helpers/Engage.pm (line 145-147) to use data-toggle:
`push @caps, sprintf '<li><a id="link-quote-%d" href="%s#quote-%d" class="text-left"  data-toggle="popover" title="%s#quote-%d"  data-content="%s#quote-%d"><i class="fa fa-fwl fa-link"></i> Link</a></li>', $post->{id}, $c->url_for, $post->{id};
}`

**Comment by jimkp** [comment]
I didn't know about the right click and copy. So maybe a way to make that obvious or the popup would work too.
